src/backend/CassandraBackend.h

```diff
@@ -271,6 +302,40 @@ class CassandraStatement
         curBindingIndex_++;
     }
 
+    // TODO: tuple binding desperately needs to be implemented using templates
+    void
```

Comment:
i don't think we should even address this TODO since we have the new framework coming up

Reply:
Yea, definitely not for the old backend.. in the new one we can implement it properly.